### PR TITLE
chore: makes app developement faster

### PR DESCRIPTION
**Pull request by redreceipt**
uses the default URL for the API so there's no need to type in the encryption password

### CODE REVIEW

**Comment by vinnyjth** [comment]
What do we do for the few PRs that rely on values stored in the `.env`? I'm thinking `OneSignal`. I'm fine merging this, but right now running commands / rebuilding the app after running `yarn app` is a bit of an undefined process.

**Comment by redreceipt** [comment]
That's true. I was thinking if there was a variable a certain needs, it would be specified the same way we do branches.

**Comment by vinnyjth** [comment]
Different idea - can we make a file called `.secret` that would be kept out of git and would store the password? We could even do the same thing on `templates` so clients would only have to create a `.secret` file with their password once, and then `yarn secrets` wouldn't need any flags.

**Comment by redreceipt** [comment]
Not sure that would solve this particular problem, the directory is wiped every run which is probably the right thing.

**Comment by vinnyjth** [comment]
The file would live on the root. Which would be true for both `yarn app` in `apps` and `yarn secrets` in templates.

**Comment by redreceipt** [comment]
Honestly not crazy about it. This is really only a problem for development in apps, don't need the password that often in templates. and it would be one more unconventional file that someone would have to explain how to use on the template. I like the current flow because if you don't know how to use it, it fails and tells you exactly what to do. 

Another reason I like this solution is that it _actually_ allows people to contribute open-source to the core app code. Up until now, unless we give out the password, you can't really contribute. I've been brainstorming how to do the same for the API, whether we have a read-only rock token or something for the API.

**Comment by vinnyjth** [comment]
fair points, how about "push enter to continue". If you didn't type a secret, default to the public api

**Comment by redreceipt** [comment]
I like that

**Comment by redreceipt** [comment]
ok @vinnyjth

File: scripts/setup-api.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ cd ..
 
 shopt -s dotglob
 mv apollos-templates/apollos-church-api/* ../ApollosApi
+cp apollos-templates/yarn.lock ../ApollosApi
```
Comment (vinnyjth) [inline scripts/setup-api.sh:22]: 👍

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**